sources/api/schnauzer/src/helpers.rs

```diff
@@ -1368,9 +1374,106 @@ enum OciSpecSection {
     Capabilities,
     ResourceLimits,
 }
-
 derive_fromstr_from_deserialize!(OciSpecSection);
```

Comment:
nit: do not delete this whitespace.

```suggestion

derive_fromstr_from_deserialize!(OciSpecSection);
```